lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -99,36 +99,36 @@
  * }
  * @psalm-type AssociationMapping = array{
  *     cache?: array,
- *     cascade?: array<string>,
+ *     cascade: array<string>,
  *     declared?: class-string,
- *     fetch?: mixed,
+ *     fetch: mixed,
  *     fieldName: string,
  *     id?: bool,
  *     inherited?: class-string,
  *     indexBy?: string,
- *     inversedBy?: string|null,
- *     isCascadeRemove?: bool,
- *     isCascadePersist?: bool,
- *     isCascadeRefresh?: bool,
- *     isCascadeMerge?: bool,
- *     isCascadeDetach?: bool,
+ *     inversedBy: string|null,
+ *     isCascadeRemove: bool,
+ *     isCascadePersist: bool,
+ *     isCascadeRefresh: bool,
+ *     isCascadeMerge: bool,
+ *     isCascadeDetach: bool,
  *     isOnDeleteCascade?: bool,
- *     isOwningSide?: bool,
+ *     isOwningSide: bool,
```

Comment:
set by default to true in _validateAndCompleteAssociationMapping (and then changed to false if needed)